regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -77,8 +77,9 @@ var ChildView = Backbone.View.extend({
     render: function() {
         this.updateWayfinding();
         this.loadImages();
-        HeaderEvents.trigger('section:open', this.id);
-        DrawerEvents.trigger('section:open', this.id);
+        // TODO: What is this for?
+        // HeaderEvents.trigger('section:open', this.id);
+        // DrawerEvents.trigger('section:open', this.id);
```

Comment:
It looks like these are listened to in `toc-view` (to highlight the appropriate section) and `sub-head-view` (to render the appropriate, human-readable label)